### ISSUE TITLE
New Outlook for Windows *Preview Release* does not currently provide third-party interoperability: OGCS won't work #1640

### ISSUE THREAD

**Issue by huotg01** [phw198/OutlookGoogleCalendarSync#1666]
This is not really a bug report. It is an information related to ticket #1640 

I don't know the value of the information, but in this post (https://lazyadmin.nl/office-365/new-outlook-for-windows/), there is this:

> No Support for COM/VSTO Add-in
> Microsoft is not planning to support COM and VSTO Add-ins at the moment. These types of add-ins are often used in corporate environments. Microsoft encourages developers to switch over to Web Add-in based on OfficeJS. The lack of support for COM and VSTO add-ins can really hold back the adoption in corporate environments.

What is "Web Add-in based on OfficeJS" ?

Gaston

**Comment by phw198** [comment]
Oh the irony - the lazy admin website's certificate is invalid :laughing: 

OfficeJS is basically "JavaScript API for Office" web-based add ons. Seems the lazy admin didn't check very far with their comment to switch over to it - there is [currently no OfficeJS Outlook API available](https://learn.microsoft.com/en-us/office/dev/add-ins/develop/application-specific-api-model) either!

However, I would agree with the comment on corporate environments - most will have line of business applications dependent on COM, and therefore won't be able to adopt the New Outlook until either Microsoft support COM, or vendors have time to rewrite their products for the JavaScript API.

It's just too early to tell....

**Comment by redfearnb** [comment]
I did try the New Outlook PWA or whatever they're going to call it. It works fairly well and didn't exactly kill OGCS working at the same time, because the same Outlook services seem to be running in the background to make the web UI version of the "New Outlook" work. However, I did start getting an error message saying multiple copies of OGCS were running; they were not.

In any case, because the "New Outlook" is still missing features I use, I switched back to it after trying it out for a couple weeks.
